test/unit/api/source/geojson.test.js

```diff
@@ -54,6 +54,28 @@ describe('api/source/geojson', () => {
             expect(source._features).toEqual([data]);
         });
 
+        it('should requestData just once', () => {
+            const data = {
+                type: 'Feature',
+                geometry: {
+                    type: 'Point',
+                    coordinates: [0, 0]
+                },
+                properties: {
+                    cartodb_id: 1
+                }
+            };
+            const source = new GeoJSON(data);
+
+            source.bindLayer(_ => _);
+            spyOn(source, '_addDataframe');
+
+            source.requestData();
```

Comment:
The test is OK. Just a suggestion. Maybe it is better in this way.

```js
expect(source._addDataframe).toHaveBeenCalledTimes(0);
source.requestData();
expect(source._addDataframe).toHaveBeenCalledTimes(1);
source.requestData();
expect(source._addDataframe).toHaveBeenCalledTimes(1);
```

Reply:
At your command!